enhancements/ingress/configurable-dns-loadbalancerservice.md

```diff
@@ -227,25 +249,35 @@ If the ingresscontroller is deleted when `.loadbalancer.dnsManagementPolicy` is
 DNS record on the cloud provider will not be deleted by the operator and will require
 the cluster admin to manually delete it. 
 
+If `.spec.domain` on the ingresscontroller does not contain the `baseDomain` of the
+cluster DNS config, then the `.loadbalancer.dnsManagementPolicy` is automatically set
+to `Unmanaged`.
+
 The support provided by the installer and this new field aren't entirely
 mutually exclusive, i.e. if customer has opted to disable DNS management cluster
 wide (as documented
 [here](https://github.com/openshift/installer/blob/master/docs/user/aws/install_upi.md#remove-dns-zones)),
 this field is of no value and need not be set since reconciliation of the *DNSRecord*
 CR is a no-op when no DNS zones are configured.
 
-__Note:__ Appropriate logging is a must on all controllers clearly indicating the
-current DNS management policy set on both ingresscontroller and DNSRecord and any
-subsequent effects, such as skipping reconciliation etc., must also be logged.
+__Note:__ 
+
+- Appropriate logging is a must on all controllers clearly indicating the
+  current DNS management policy set on both ingresscontroller and DNSRecord and any
+  subsequent effects, such as skipping reconciliation etc., must also be logged.
+- Automatic `dnsManagementPolicy` assignment based on the `domain` is currently only
+  supported on the AWS Platform. It will be expanded to other clouds once we are sure
+  no one is depending on this behaviour ([BZ#2041616](https://bugzilla.redhat.com/show_bug.cgi?id=2041616)
+  for additional information).
 
 ### Risks and Mitigations
 
-This feature does not support default ingresscontrollers, if updated to be unmanaged
-in conjunction with updating the domain could result in breaking cluster connectivity
-if not all components are properly updated. This should be done at the discretion
-of the cluster admin. The manual creation of the required DNS records will need to be
-done prior to making this change, as the canary controller will try to ensure that the
-ingress domains are reachable, and otherwise this sets the `cluster-ingress-operator`
+This feature does not support default ingresscontrollers when the ingresscontroller is 
+updated to be unmanaged in conjunction with updating the domain could result in breaking
```

Comment:
```suggestion
This feature does not support default ingresscontrollers when the ingresscontroller is 
updated to be unmanaged in conjunction with updating the domain, which could result in breaking
```
or
```suggestion
This feature does not in itself provide a means to change the default ingresscontroller's
domain. Even if the cluster admin were to change the ingresscontroller's `spec.domain`
and then use this feature to configure a custom DNS record for the new domain, changing
the domain will break
```
Maybe this should be listed as a non-goal:  "Enable the cluster admin to change the default ingresscontroller's domain (which would also require updating routes and certificates)."